docs/apis/api-overview.md

```diff
@@ -5,4 +5,8 @@ sidebar_label: Overview APIs and Clients
 description: "This section contains information about the different APIs and Clients to use with Miranum."
 ---
 
```

Comment:
API Overview instead of "Overview of the APIs" ?

Reply:
That sounds better indeed!